src/Data/Gdpr/GdprEditingConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { provideEditingConfig } from 'scrivito'
+import { Gdpr } from './GdprDataClass'
+
+provideEditingConfig(Gdpr, {
+  title: 'GDPR',
```

Comment:
* [ ] FYI The items are called "GDPR", but GDPR is a single regulation, how can there be many? How about "GDPR consent"?

Reply:
I also updated the wording of the content.